tests/test_transfer.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from ioSPI import transfer
+
+
+def test_ctf_freqs():
+    """Test for ctf_freq."""
+    N = (np.random.randint(low=8, high=128) // 2) * 2
+    freq_1d = transfer.ctf_freqs(N, d=1)
+    assert freq_1d.shape == (N // 2,)
+    assert np.isclose(freq_1d.min(), 0)
+    assert np.isclose(freq_1d[0], 0)
+    assert freq_1d[-1] < N // 2
+    psize = np.random.uniform(low=1, high=10)
+    freq_A_2d, angles_rad = transfer.ctf_freqs(N, d=2, psize=psize)
+    assert np.isclose(freq_A_2d[N // 2, 0], 0.5 * psize)
+    assert np.isclose(freq_A_2d[N // 2, N // 2], 0)  # dc
+    assert freq_A_2d.shape == angles_rad.shape == (N, N)
+    assert (-np.pi <= angles_rad.min()) and (angles_rad.max() <= np.pi)
+
+
+def test_eval_ctf():
+    """Test for eval_ctf."""
+    N = (np.random.randint(low=8, high=128) // 2) * 2
```

Comment:
lowercase

Reply:
There are many places where I use `N` to refer to the length along one axis in an array (pixels, voxels, etc). Should I change them all. I thought it was OK because it was a constant. Here, in this test, I'm just using a random integer.

Reply:
ok I see you said n_pixels above. I'll do that everywhere.